sections/main-article.liquid

```diff
@@ -278,7 +278,7 @@
         "logo": {
           "@type": "ImageObject",
           "height": {{ settings.share_image.height | json }},
-          "url": {{ settings.share_image | image_url: width: settings.share_image.width | prepend: "https:" | json }},
+          "url": {{ settings.share_image | image_url: width: 1920 | prepend: "https:" | json }},
           "width": {{ settings.share_image.width | json }}
```

Comment:
I'm not sure for 100% but I can assume that `"height":` and `"width":` may lead to an error. I would remove them, or leave only one of them with fixed value. What do you think?

Reply:
I'm not sure I follow. 
>or leave only one of them with fixed value

Isn't it what I'm doing in the PR already ^ ?

As for adding multiple images, yeah I think it could be done as a follow up if we see it's necessary 👍

Reply:
I'm sorry Ludo, it seems like I'm bad at explaining. My concern is next:

- line 281 - you provide `url` with fixed `width: 1920`.
- line 282 - you provide `width` with value **`settings.share_image.width`** that may pass an original size of an image and it can be 5000px or bigger.

So, there shouldn't be an error in Liquid as it was before this PR but I just wanna be sure that there is no conflict or potential issue for merchants for their search optimization. Because I don't know what happens in case when the actual image is pulled through the url has **`width 1920px`**, but at the same time we are telling Schema.org to use `width: settings.share_image.width` which can be 5000px for example.

Reply:
Oh I see what you mean 👍 good point, thanks for bringing it up 🙂